Sources/SwiftProtobuf/JSONEncodingOptions.swift

```diff
@@ -18,5 +18,8 @@ public struct JSONEncodingOptions {
   /// Always print enums as ints. By default they are printed as strings.
   public var alwaysPrintEnumsAsInts: Bool = false
 
+  /// Always print proto field name instead of JSON(lowerCamelCase) name. By default they are printed as JSON name.
+  public var alwaysPrintProtoFieldNames: Bool = false
```

Comment:
`alwaysPrint` doesn't seem right here, as it seems like if you weren't print the field names you'd maybe use the field numbers instead?

The C++ seems to call this [`preserve_proto_field_names`](https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/util/json_util.h#L81) ([CHANGES.txt entry](preserve_proto_field_names))

So maybe some naming more along those lines so it better indicates it is picking between the two string forms?